packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1450,49 +1559,27 @@ export class Combobox
     });
   }
 
-  private renderAllSelectedIndicatorChip(): JsxNode {
+  private renderAllSelectedIndicatorChip(compact: boolean): JsxNode {
     const {
       compactSelectionDisplay,
       scale,
       selectedVisibleChipsCount,
       setAllSelectedIndicatorChipEl,
     } = this;
-    const label = this.messages.allSelected;
-    return (
-      <calcite-chip
-        class={{
-          chip: true,
-          [CSS.chipInvisible]: !(
-            this.isAllSelected() &&
-            !selectedVisibleChipsCount &&
-            !compactSelectionDisplay
-          ),
-        }}
-        label={label}
-        ref={setAllSelectedIndicatorChipEl}
-        scale={scale}
-        title={label}
-        value=""
-      >
-        {label}
-      </calcite-chip>
-    );
-  }
+    const label = compact ? this.messages.all || "All" : this.messages.allSelected;
```

Comment:
fixme: I don't think we need the `"All"` fallback here. We don't do text fallbacks in other components.